sklearn/dummy.py

```diff
@@ -355,6 +354,15 @@ def score(self, X, y, sample_weight=None):
             X = np.zeros(shape=(len(y), 1))
         return super().score(X, y, sample_weight)
 
+    @deprecated(
+        "The outputs_2d_ attribute is deprecated in version 0.22 "
+        "and will be removed in version 0.24. It is equivalent to "
+        "n_outputs_ != 1."
```

Comment:
Maybe `n_outputs_ > 1` would be slightly better? I see that, `n_output == 0` is not possible in the code, but still the above comparison suggests that even that case would  correspond to to `output_2d_ is True` which is not the case.